docs/configuration/flag_configuration.md

```diff
@@ -103,43 +162,133 @@ The output of the targeting rule **must** match the name of one of the variants
 If an invalid or null value is is returned by the targeting rule, the `defaultVariant` value is used.
 If no targeting rules are defined, the response reason will always be `STATIC`, this allows for the client side caching of these flag values, this behavior is described [here](../other_resources/caching.md).
 
-The [JSON Logic playground](https://jsonlogic.com/play.html) is a great way to experiment with new targeting rules.
-The following example shows how a rule could be configured to return `binet` when the email (which comes from evaluation context) contains `@faas.com`.
-If the email wasn't included in the evaluation context or doesn't contain `@faas.com`, null is returned and the `defaultVariant` is used instead.
-
-<details>
-  <summary>Click here to see how this targeting rule would look in the JSON Logic playground.</summary>
-
-1. Open the [JSON Logic playground](https://jsonlogic.com/play.html) in your favorite browser
-1. Add the follow JSON as the `Rule`:
-
-    ```json
-    {
-      "if": [
-        {
-          "in": [
-            "@faas.com",
-            {
-              "var": ["email"]
-            }
-          ]
-        },
-        "binet",
-        null
-      ]
-    }
-    ```
+#### Evaluation Context
```

Comment:
I think we should explain what evaluation context means. Highlighting that this comes from flag evaluation request body. And also provide an example to connect with variables defined below, 

```
{
    "flagKey":"booleanFlagKey",
    "context":{
        "email": "abc@abc.com"
    }
}
```